schemas/api.yml

```diff
@@ -18,17 +18,25 @@ info:
 
     # Marketplace applications development
     To develop Marketplace application you need to create an application in Uspacy Marketplace. You can do it via request to [Uspacy support](mailto:support@uspacy.com).
-    You will receive ```client_id``` and ```client_secret``` for your application. You will also need to provide ```callback_uri``` for your application.
-    After user installs your application you will receive POST request with json in callback. It will contain ```jwt``` token and ```refreshToken```.
+    You will receive ```client_id``` and ```client_secret``` for your application. You will also need to provide ```callback_url``` for your application.
+    After that, the install/uninstall endpoints for your applications should be implemented. Please note that the path for these endpoints should follow this format:
+    ```callback_url/portals/install``` and ```callback_url/portals/uninstall```, respectively.
```

Comment:
make  `{{callback_url}}/portals/install` – to show that is a template
and add an example as well, like if your callback_url is https://google.com/handler/v1, we will send  webhook to https://google.com/handler/v1/portals/install on install and https://google.com/handler/v1/portals/uninstall